packages/url_launcher/url_launcher_web/pubspec.yaml

```diff
@@ -12,18 +12,17 @@ flutter:
         fileName: url_launcher_web.dart
 
 dependencies:
+  url_launcher_platform_interface: ^1.0.1
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
   meta: ^1.1.7
-  url_launcher_platform_interface: ^1.0.1
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  url_launcher:
-    path: ../url_launcher
+  url_launcher: ^5.2.5
```

Comment:
Does `5.2.5` work with web? The build dashboard had to do a manual set to get it working again (see https://github.com/flutter/cocoon/pull/517)

Reply:
I've looked at the errors that prompted your change, and it seems that you attempted to use this package (too) early on, when it still used Method Channels. The most current implementations of this plugin and `url_launcher` shouldn't use method channels in the browser, and instead use the platform interface. Please, give this version of the plugin a shot when it gets published!

Reply:
Thanks David! I opened a PR to update the build dashboard back to using the latest then :)